internal/search/query/parser.go

```diff
@@ -35,6 +35,7 @@ type Parameter struct {
 	Field   string `json:"field"`   // The repo part in repo:sourcegraph.
 	Value   string `json:"value"`   // The sourcegraph part in repo:sourcegraph.
 	Negated bool   `json:"negated"` // True if the - prefix exists, as in -repo:sourcegraph.
+	Quoted  bool   `json:"quoted"`  // True if the parsed value was quoted
```

Comment:
```suggestion
	Quoted  bool   `json:"quoted"`  // True if the parsed value was quoted.
```

Only because I see other comments have a period, just for consistency and cure my OCD (Obsessive-compulsive disorder) :)